File: Cristian_viana_camacho/programas/poo/actividad.dart
class Habitacion {
  int? metrosCuadrados;
  String? muebles;
  String? color;

}

class Consola {
  String? marca;
  String? color;
  int? lanzamiento;

}

class Multimedia {
  String? tipo;
  String? genero;
  int? duracion;

}

void main() {

  Habitacion estudio = new Habitacion();
  estudio.metrosCuadrados = 30;
  estudio.muebles = "escritorios";
  estudio.color = "gris y blanco";

  Consola playStation = new Consola();
  playStation.marca = "sony";
  playStation.color = "blanco";
  playStation.lanzamiento = 2020;

  Multimedia callOfDuty = new Multimedia();
  callOfDuty.tipo = "juego";
  callOfDuty.genero = "FPS";
  callOfDuty.duracion = 420;

  print(callOfDuty.tipo);
  print(playStation.marca);
  print(estudio.muebles);
}

File: Cristian_viana_camacho/programas/poo/actividad.rb
class Habitacion 
  attr_accessor :metrosCuadrados, :muebles, :color
end

class Consola 
    attr_accessor :marca, :color, :lanzamiento
end

class Multimedia 
    attr_accessor :tipo, :genero, :duracion
end



  estudio = Habitacion.new
  estudio.metrosCuadrados = 30
  estudio.muebles = "escritorios"
  estudio.color = "gris y blanco"

  playStation = Consola.new
  playStation.marca = "sony"
  playStation.color = "blanco"
  playStation.lanzamiento = 2020

  callOfDuty = Multimedia.new
  callOfDuty.tipo = "juego"
  callOfDuty.genero = "FPS"
  callOfDuty.duracion = 420

  puts "#{callOfDuty.tipo}"
  puts "#{playStation.marca}"
  puts "#{estudio.muebles}"
